### PR TITLE
Add 2023 minimum eff. ASHPs to options_lookup.tsv

### PR DESCRIPTION
**Pull request by whiphi92**
## Pull Request Description

Adding 2023 minimum efficiency air-source heat pumps (ASHPs) to the resources/options_lookup.tsv file. 

For northern states: 14 SEER 8.8 HSPF
For southern states: 15 SEER 8.8 HSPF
source: https://www.eia.gov/todayinenergy/detail.php?id=40232

Eric Wilson (@ejhw) and I thought it would be a good idea to add these ASHPs to the options_lookup.tsv file since they will be common HVAC units come 2023. I determined COPs by interpolating between HSPFs and COPs of existing ASHP options.

## Checklist

Not all may apply:

- [ ] Unit tests have been added or updated
- [ ] All rake tasks have been run, and pass
- [ ] Documentation has been modified appropriately
- [ ] Any new options are added to `project_testing`
- [ ] `project_testing` runs without any failures
- [ ] No unexpected regression test changes
- [ ] All tests are passing
- [ ] The [changelog](https://github.com/NREL/resstock/blob/master/CHANGELOG.md) has been updated appropriately
- [ ] This branch is up-to-date with develop

For more information on how to perform these checklist items, see the documentation's [Advanced Tutorial](https://resstock.readthedocs.io/en/latest/advanced_tutorial/index.html).

### CODE REVIEW

**Review by ejhw — CHANGES_REQUESTED** [review]
Shouldn't COP be the same for these two options since HSPF is the same? I think COP is for heating only (EER is the equivalent argument for SEER).

**Comment by whiphi92** [comment]
> 
> 
> Shouldn't COP be the same for these two options since HSPF is the same? I think COP is for heating only (EER is the equivalent argument for SEER).

You're right. Both cop values should be 3.68.

**Comment by ejhw** [comment]
@whiphi92  Sorry to change my mind on this, but I checked the ENERGY STAR listing and didn't see any units with HSPF 8.8, only an even 9.0. I also didn't see any SEER 14, HSPF 9.0 combinations. SEER 15, HSPF 9.0 seemed to be fairly common, so I'd suggest we just add a single new option that is SEER 15, HSPF 9.0.

**Comment by whiphi92** [comment]
Just edited the resources/options_lookup.tsv to only have the 15 SEER 9.0 HSPF ASHP.

**Review by ejhw — APPROVED** [review]

File: resources/options_lookup.tsv [hunk 1]
```diff
@@ -699,6 +699,9 @@ HVAC Heating Efficiency	"ASHP, SEER 14.3, 8.5 HSPF"	ResidentialHVACAirSourceHeat
...
 		ResidentialAirflow																																													
 		ResidentialHVACSizing																																													
+HVAC Heating Efficiency	"ASHP, SEER 15, 8.8 HSPF"	ResidentialHVACAirSourceHeatPumpSingleSpeed	seer=15.0	hspf=9.0	eer=12.7	cop=3.8	shr=0.73	fan_power_rated=0.365	fan_power_installed=0.5	min_temp=0.0	crankcase_capacity=0.02	crankcase_temp=55.0	eer_capacity_derate_1ton=1.0	eer_capacity_derate_2ton=1.0	eer_capacity_derate_3ton=1.0	eer_capacity_derate_4ton=1.0	eer_capacity_derate_5ton=1.0	cop_capacity_derate_1ton=1.0	cop_capacity_derate_2ton=1.0	cop_capacity_derate_3ton=1.0	cop_capacity_derate_4ton=1.0	cop_capacity_derate_5ton=1.0	heat_pump_capacity=autosize	supplemental_efficiency=1.0	supplemental_capacity=autosize	dse=NA
```
Comment (ejhw) [inline resources/options_lookup.tsv:702]: Option name HPSF needs to be updated.

**Comment by ejhw** [comment]
@shorowit @jonwinkler FYI, We're adding a "ASHP, SEER 15, 9.0 HSPF" option to options_lookup. It is a copy of "ASHP, SEER 15, 8.5 HSPF" with the HSPF and COP changed. Let us know if you see any problems with doing that. Thanks.

File: resources/options_lookup.tsv (hunk 1 above)
Comment (whiphi92) [reply resources/options_lookup.tsv:702]: Updated in [76e67de](https://github.com/NREL/resstock/pull/557/commits/76e67deb569f472111b96f0ebdb8ce61125db85b).

**Review by ejhw — APPROVED** [review]

**Comment by whiphi92** [comment]
Creating a new PR from internal branch, not external.

**Pull request closed without merging**